src/main/scheduler/scheduler.c

```diff
@@ -490,6 +494,14 @@ FAST_CODE void scheduler(void)
                 taskExecutionTimeUs += schedulerExecuteTask(getTask(TASK_PID), currentTimeUs);
             }
 
+            // Check for failsafe conditions without reliance on the RX task being well behaved
+            if (millis() - lastFailsafeCheck > PERIOD_RXDATA_FAILURE) {
```

Comment:
`cmp32_t` or `cmpTimeUs` with `micros()`. IMO micros is preferable, reusing `schedulerStartTimeUs`

Reply:
See above. Also `mills()` is very low cost.